packages/ckb-sdk-utils/package.json

```diff
@@ -32,6 +32,7 @@
   },
   "dependencies": {
     "@nervosnetwork/ckb-types": "0.36.0",
+    "blake2b": "^2.1.3",
```

Comment:
Don't use range version

Reply:
the version is modified to a fixed version